doc/uml/Class_Diagram

```diff
@@ -0,0 +1,49 @@
+@startuml
+class Data{
+-Idenitifer
+-Content
+}
+
+class Tag{
+-Identifier
+-Sortable Ordinal
++rename()
+}
+
+class Database{
+-Tags[]
+-Data[]
+-Relationships[]
++createData()
++deleteData()
++createTag()
++deleteTag()
+}
+
+class Relationship{
+-Relationship(Data, Tag)
++returnTag()
++returnData()
++changeRelationship()
```

Comment:
Get rid of this. We can effectively change a relationship just through create/delete; the relationship doesn't do much other than connect data/tag so it's better viewed through an immutable light.